src/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  width: 33%;
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+}
+
+.block-red {
+  background-color: red;
+  order: 1;
+}
+
+.block-red80 {
+  background-color: #c00;
+  order: 2;
+}
+
+.block-red60 {
+  background-color: #900;
+  order: 3;
+}
```

Comment:
you could not provide `order` value to elements, that appears in default order. By default it's set to 0, and they will display in order, that they described in html